soc/arm/microchip_mec/mec1501/Kconfig.soc

```diff
@@ -16,6 +16,10 @@ config SOC_MEC1501_HSZ
 
 endchoice
 
+config RTOS_TIMER
+	bool "MEC1501 RTOS timer"
+	default n
```

Comment:
Better to remove this `default n`. See https://docs.zephyrproject.org/latest/guides/kconfig/index.html#redundant-defaults.

Reply:
This commit was mistakenly added into this branch. 
KConfig changes are part of this PR https://github.com/zephyrproject-rtos/zephyr/pull/17898
I have addressed your comment there.